sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/campaigns/CampaignDashboardDiagramComponent.java

```diff
@@ -63,6 +65,7 @@ public CampaignDashboardDiagramComponent(
 			totalValuesWithoutStacks = true;
 		}
 
+		showAsColumnChart = DiagramType.values().length > 0 && DiagramType.COLUMN == DiagramType.values()[0];
```

Comment:
This solution is ok for switching but would fail if i want to define the default diagram type as 'bar', please extend the enum type and make sure a diagram can be by default bar type

Reply:
Okay, agree